aeron-system-tests/src/test/java/io/aeron/cluster/ClusterTest.java

```diff
@@ -1975,8 +1984,14 @@ void shouldTrackSnapshotDuration()
         cluster.awaitSnapshotCount(1);
         cluster.awaitNeutralControlToggle(leader);
 
-        assertEquals(1L, snapshotDurationThresholdExceededCount.get());
-        assertTrue(maxSnapshotDuration.get() > 0);
+        assertEquals(1L, totalSnapshotDurationTracker.snapshotDurationThresholdExceededCount().get());
+        assertTrue(totalSnapshotDurationTracker.maxSnapshotDuration().get() > 0);
+
+        assertEquals(1L, service1SnapshotDurationTracker.snapshotDurationThresholdExceededCount().get());
+        assertTrue(service1SnapshotDurationTracker.snapshotDurationThresholdExceededCount().get() > 0);
```

Comment:
The second `assertTrue` is redundant as it checks the same thing as the previous line.
Did you intend to check the `maxSnapshotDuration` instead?

Reply:
fixed